From a5209418ca08d5c1f3f7681285f3167420574cb3 Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <Dinh.Nguyen@freescale.com>
Date: Fri, 10 Sep 2010 15:44:55 -0500
Subject: [PATCH] ENGR00127383 usb: enable usb remote wake up for all ports

Enable remote wakeup for all ports that support remote
wakeup.

Signed-off-by: Dinh Nguyen <Dinh.Nguyen@freescale.com>
---
 arch/arm/plat-mxc/usb_common.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/usb_common.c b/arch/arm/plat-mxc/usb_common.c
index 0a94d72..a13b2c7 100644
--- a/arch/arm/plat-mxc/usb_common.c
+++ b/arch/arm/plat-mxc/usb_common.c
@@ -449,6 +449,7 @@ static int usb_register_remote_wakeup(struct platform_device *pdev)
 	}
 	irq = res->start;
 	pdev->dev.power.can_wakeup = 1;
+	device_set_wakeup_enable(&(pdev->dev), 1);
 	enable_irq_wake(irq);
 
 	return 0;
-- 
1.5.4.4

